.gitignore

```diff
@@ -22,3 +22,5 @@ config.env
 .npm
 
 .DS_Store
+
+ youtube.md
```

Comment:
I've now deleted the .gitignore file and created a new one where I added the extra stuff I wanted to be ignored. That youtube.md shouldn't have been pushed at all.